rust/benches/scan.rs

```diff
@@ -15,6 +15,17 @@
 // specific language governing permissions and limitations
 // under the License.
 
+//! Before running the dataset, prepare a "test.lance" dataset, in the
```

Comment:
how to prepare though? python? rust script?

Reply:
You should use either python/rust to generate the dataset. It is also a good test for guaranteeing compatibility between implementations.